packages/serverpod_client/lib/src/method_stream/method_stream_connection_details.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+
+/// Details for a method stream connection.
+class MethodStreamConnectionDetails {
+  /// The endpoint to connect to.
+  final String endpoint;
+
+  /// The method to call.
+  final String method;
```

Comment:
Not sure that it is required. Since it is a string it could be nothing but the endpoint name. But I understand the point of using the same naming convention across the board.